### PR TITLE
[vividus] Add WordUtils support in eval expression

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

File: vividus/src/main/java/org/vividus/bdd/expression/EvalExpressionProcessor.java [hunk 1]
```diff
@@ -36,7 +37,8 @@
     private static final Pattern EVAL_PATTERN = Pattern.compile("^eval\\((.*)\\)$", Pattern.CASE_INSENSITIVE
             | Pattern.DOTALL);
     private static final int EVAL_GROUP = 1;
-    private static final Map<String, Object> NAMESPACES = Map.of("math", Math.class, "stringUtils", StringUtils.class);
+    private static final Map<String, Object> NAMESPACES = Map.of("math", Math.class, "stringUtils", StringUtils.class,
+            "wordUtils", WordUtils.class);
```
Comment (valfirst) [inline vividus/src/main/java/org/vividus/bdd/expression/EvalExpressionProcessor.java:41]: I doubtful we need to add more and more utils here, what about just new simple expression?

File: docs/modules/commons/pages/expressions.adoc [hunk 2]
```diff
@@ -252,6 +252,44 @@ When I initialize the scenario variable `my-data` with value `#{resourceToBase64
...
+----
+#{eval(*$script*)}
+----
+
+* *`$script`* - valid https://commons.apache.org/proper/commons-jexl/reference/syntax.html[JEXL] script to be evaluated
+
+[TIP]
+====
+* Any Vividus variable is accessible in the JEXL script by its name
+====
```
Comment (valfirst) [inline docs/modules/commons/pages/expressions.adoc:269]:
```suggestion
TIP: Any Vividus variable is accessible in the JEXL script by its name
```

File: docs/modules/commons/pages/expressions.adoc [hunk 3]
```diff
@@ -252,6 +252,44 @@ When I initialize the scenario variable `my-data` with value `#{resourceToBase64
...
+
+
+.Evaluate JEXL script
+[source,gherkin]
+----
+Scenario: Verify eval expression
+Then `#{<expression>}` is = `<expected>`
+Examples:
+|expected          |expression                                                        |
```
Comment (valfirst) [inline docs/modules/commons/pages/expressions.adoc:278]: change columns order to make it more logical

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus/src/main/java/org/vividus/bdd/expression/EvalExpressionProcessor.java (hunk 1 above)
Comment (ikalinin1) [reply vividus/src/main/java/org/vividus/bdd/expression/EvalExpressionProcessor.java:41]: Why not?
I see only one problem regarding the syntax of such logic, but for me, flexibility of the code worth it

**Review by valfirst — APPROVED** [review]

**Review by abudevich — APPROVED** [review]

**Pull request merged**